reseq_ros2/reseq_ros2/communication.py

```diff
@@ -220,16 +182,21 @@ def topic_from_name(self, name: str) -> rc.ReseQTopic:
 
 def main(args=None):
```

Comment:
I have modified the function in a way that I think will not cause the misleading information (process finished cleanly) to appear in the log file.